src/ferc_xbrl_extractor/taxonomy.py

```diff
@@ -230,8 +230,8 @@ class Taxonomy(BaseModel):
     @classmethod
     def from_path(
         cls,
-        path: str,
-        archive_file_path: str | None = None,
+        path: Path,
+        archive_path: Path | None = None,
```

Comment:
Rename for consistency with the command-line arg, but one day it'd be nice to do away with this weird double-path thing.

Reply:
We can probably deduce the path to the entry point and not pass this, but probably something we can leave for the future.